app/models/box.rb

```diff
@@ -94,4 +94,12 @@ def blind_attribute?(attr_name)
   def transferred?
     box_transfers.exists?
   end
+
+  def unblind!
+    self.update(blinded: false)
```

Comment:
`self` is implicit in Ruby, you don't have to specify it (same in `#blind!`). You may also want to use [#update_columns](https://api.rubyonrails.org/v6.1/classes/ActiveRecord/Persistence.html#method-i-update_columns) which would be a bit faster (no need to re-validate the box, we only change one value):

```suggestion
    update_columns(blinded: false)
```

Same for `#blind!`